src/main/java/com/exasol/projectkeeper/validators/changelog/DependencySectionFixer.java

```diff
@@ -0,0 +1,85 @@
+package com.exasol.projectkeeper.validators.changelog;
+
+import static com.exasol.projectkeeper.validators.changelog.ChangelogFile.DEPENDENCY_UPDATES_HEADLINE;
+
+import java.io.*;
+import java.nio.file.Path;
+import java.util.*;
+
+import org.apache.maven.model.Build;
+import org.apache.maven.model.Model;
+import org.apache.maven.model.building.ModelBuildingException;
+import org.codehaus.plexus.util.xml.pull.XmlPullParserException;
+
+import com.exasol.errorreporting.ExaError;
+import com.exasol.projectkeeper.validators.changelog.dependencies.*;
+
+/**
+ * This class fixes the dependency section of a {@link ChangelogFile}.
+ */
+class DependencySectionFixer {
+    private final Path projectDirectory;
+    private final Model currentMavenModel;
+
+    /**
+     * Create a new instance of {@link DependencySectionFixer}.
+     * 
+     * @param projectDirectory projects root directory
+     */
+    public DependencySectionFixer(final Path projectDirectory) {
```

Comment:
Can you inject here current model and model from the latest release? This will help separating parsing files and fixing

Reply:
I used dependency injection for `DependencyChangeReportReader`. I designed this as the class that assembles the components.
Wat would be reasons for a dependency injection here?